src/main/resources/application.yaml

```diff
@@ -1,6 +1,12 @@
 server:
   port: 4000
 
+logging:
+  level:
+    org:
+      springframework:
+        security: DEBUG
```

Comment:
I suppose that is temporary and will be removed.

Reply:
yes, it will